samples/SemanticKernel/Encamina.Enmarcha.Samples.SemanticKernel.QuestionAnswering/Encamina.Enmarcha.Samples.SemanticKernel.QuestionAnswering.csproj

```diff
@@ -8,15 +8,15 @@
   </PropertyGroup>
 
     <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-        <NoWarn>1701;1702;SKEXP0003;SKEXP0011;SKEXP0052;</NoWarn>
+        <NoWarn>1701;1702;SKEXP0001;SKEXP0003;SKEXP0010;SKEXP0011;SKEXP0050;SKEXP0052;</NoWarn>
     </PropertyGroup>
 
     <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-        <NoWarn>1701;1702;SKEXP0003;SKEXP0011;SKEXP0052;</NoWarn>
+        <NoWarn>1701;1702;SKEXP0001;SKEXP0003;SKEXP0010;SKEXP0011;SKEXP0050;SKEXP0052;</NoWarn>
```

Comment:
Ídem.

Check this exceptions, I don't know if all of them are still valid or required since the SK release notes indicates the following → «_Reduce number of experimental flags_»

This means that perhaps some exceptions we had are not required anymore. 

Please verify this in every project that has been affected.

Reply:
Remove `SKEXP0003`, `SKEXP0011` and `SKEXP0052` exceptions.